hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/auth/ITestAssumeRole.java

```diff
@@ -524,78 +521,6 @@ public Path methodPath() throws IOException {
     return path(getMethodName());
   }
 
-  @Test
-  public void testRestrictedRename() throws Throwable {
-    describe("rename with parent paths not writeable");
-    executeRestrictedRename(createAssumedRoleConfig());
-  }
-
-  @Test
-  public void testRestrictedSingleDeleteRename() throws Throwable {
```

Comment:
why do you remove old tests? just asking because of backward compatibility; maybe makes sense.

Reply:
moved to ITestPartialRenamesDeletes and extended, parameterized etc.